### PR TITLE
Fr p1 profile page

### PR DESCRIPTION
**Pull request by elp49**
Capture

responsiveness is still in progress

### CODE REVIEW

File: frontend/src/components/ProfileContent.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    username: string;
+    name: string;
+    profileImg: string;
+    gender: string;
+    religion: string;
+    location: string;
+    occupations: string[];
+    age: number | string,
+    interests: string[];
+};
```
Comment (mneedleman) [inline frontend/src/components/ProfileContent.tsx]: I think this needs a more descriptive name than `Props`. Why not `ProfileContentProps`?

File: frontend/src/components/ProfileContent.tsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    },
+    field: {
+        fontSize: '2rem',
+        padding: 5,
+        paddingRight: 10,
+    },
+});
+
+export default function ProfileContent(props: Props) {
+    const classes = useStyles();
```
Comment (mneedleman) [inline frontend/src/components/ProfileContent.tsx]: I really like that you decided to pull this out into a separate component! It'll definitely come in handy later.

File: frontend/src/components/ProfileContent.tsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                <Grid container item alignItems="center" justify="center" xs={6}>
+                    <Avatar alt={props.name} src={props.profileImg} className={classes.profileAvatar} />
+                </Grid>
+                <Grid container item alignItems="center" justify="center" xs={6}>
+                    <div className={classes.introVideo}>Intro Video</div>
+                </Grid>
+            </Grid>
+            <Grid container item alignItems="center" className={classes.fieldsContainer} xs={12}>
+                {profileFields.map(({ title, value }: ProfileField) => (
+                    <Grid key={title} item className={classes.field} sm={12} md={6}>
```
Comment (mneedleman) [inline frontend/src/components/ProfileContent.tsx:106]: Great work!

File: frontend/src/components/ProfileSidebar.tsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+import NotificationsIcon from '@material-ui/icons/Notifications';
+import SettingsIcon from '@material-ui/icons/Settings';
+import MailIcon from '@material-ui/icons/Mail';
+import LockIcon from '@material-ui/icons/Lock';
+
+// ProfileSidebar props.
+type Props = {
+    name: string;
+    profileImg: string;
+}
```
Comment (mneedleman) [inline frontend/src/components/ProfileSidebar.tsx]: I think this should have a more descriptive name than `Props`. Why not `ProfileSidebarProps`?

**Review by mneedleman — CHANGES_REQUESTED** [review]
This is really good! Just a couple small changes I'd like to see and it's good to go. 👍

**Comment by elp49** [comment]
requested code changes completed

**Review by mneedleman — APPROVED** [review]
Great!

**Pull request merged**